src/pages/pokemon-page/pokemon-page.component.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import './pokemon-page.styles.scss';
+
+class PokemonPage extends React.Component {
+  render() {
+    const { freePokemons, capturedPokemons, match } = this.props;
+    const pokemonID = +match.params.pokemonID;
+    let currentPokemon =
+      freePokemons.filter(pokemon => pokemon.id === pokemonID)[0] ||

```

Comment:
just to add field to the store `currentPokemon` and use it instead of iterating through all pokemons